multiversx_sdk_cli/contracts.py

```diff
@@ -90,7 +93,9 @@ def prepare_deploy_transaction(self,
                                    version: int,
                                    options: int,
                                    guardian: str) -> Transaction:
-        args = prepare_args_for_factory(arguments) if arguments else []
+        args = arguments if arguments else []
+        if not args_from_file:
```

Comment:
Maybe rename to `should_prepare_args` or `are_args_already_prepared` or something similar?

Reply:
renamed to `should_prepare_args`.